### ISSUE TITLE
Registration error message title perhaps?

### ISSUE THREAD

**Issue by adamsilver** [adamsilver/f#58]
Whether this text is to say ‘Current page’, ‘This page’ or ‘You are here’ is somewhat open to debate, but it should be fairly succinct. In any case, I've preprended the <span> so the screen reader says blah rathan than blah.
